shears/mine_for_gold.py

```diff
@@ -15,7 +18,10 @@
     '(by /u/projectmerino)'
 )
 
+session = Session()
+session.verify = "/path/to/certfile.pem" 
 
+""" Note: If you are only analyzing public comments, entering a username and password is optional. """
```

Comment:
Docstrings should be inside function definitions: https://www.python.org/dev/peps/pep-0257/